fdbmonitor/fdbmonitor.cpp

```diff
@@ -967,7 +979,7 @@ void watch_conf_file( int kq, int* conff_fd, const char* confpath ) {
 	}
 
 	/* Open and watch */
-	*conff_fd = open( confpath, O_EVTONLY );
+	*conff_fd = open( confpath, O_RDONLY );
```

Comment:
Above, the code was duplicated to avoid changing `O_EVTONLY` to `O_RDONLY`, but then here it was just changed for all platforms.  Would it be easier to just `#define O_EVTONLY O_RDONLY` if FreeBSD, or create a new define flag to use based on platform to avoid duplication and to avoid changing this on anything other than FreeBSD?

Reply:
addressed in next push